package.json

```diff
@@ -4,26 +4,30 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js",
+    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js --timeout 45000 --exit",
     "coverage": "nyc report --reporter=text",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "lint": "eslint \"**/*.js\" --ignore-pattern node_modules/ --fix",
     "build": "babel ./src -d dist",
     "start": "node dist/index.js",
-    "start:dev": "nodemon --exec babel-node -- src/index.js",
+    "sync": "DEBUG=dev babel-node ./src/database/reSync",
+    "start:dev": "npm run sync && nodemon --exec babel-node -- src/index.js",
     "migrate:dev": "sequelize db:migrate",
     "migrate:test": "cross-env NODE_ENV=test sequelize db:migrate",
     "migrate:prod": "cross-env NODE_ENV=production sequelize db:migrate",
     "undo:dev": "sequelize db:migrate:undo",
     "undo:test": "cross-env NODE_ENV=test sequelize db:migrate:undo",
-    "undo:prod": "cross-env NODE_ENV=prooduction sequelize db:migrate:undo",
+    "undo:prod": "cross-env NODE_ENV=production sequelize db:migrate:undo",
```

Comment:
Why add the `undo:prod` script? This should never be done, you wouldn't want to have a script that can drop your production database readily available to people, it can be extremely costly.

Reply:
@marusoft @wahaaabello @dewaleolaoye @Dolapo47

Reply:
@dewaleolaoye this is for you. This aspect of the code was pulled from stage merged from your merge.